### ISSUE TITLE
Set placeholder color explicitly with SCSS 

### ISSUE THREAD

**Issue by jackkoppa** [politico/vue-accessible-selects#80]
We should be explicitly setting the `color` of the placeholder text for `SelectSingle` and `SelectMulti`; currently it just relies on the browser default, leading to color contrast issues.

There's no real way to "enforce" this color contrast behavior for the placeholder text (because it's just SCSS that a user will set elsewhere, we can't validate it using JS - note that this is probably an argument for using CSS variables), but we should still be at least setting it explicitly based on a default value.

**Comment by jackkoppa** [comment]
Let's allow the user to pass in their own `$placeholder-color` and add a `$default-placeholder-color`

https://github.com/politico/vue-accessible-selects/blob/main/src/styles/mixins.scss#L2

**Comment by hungsu** [comment]
This is almost complete. @ivanshtyrlyaev has added a default placeholder colour, but this default should be our VulcanGray47, or hsl(224, 8%, 47%), or #6e7381
